src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -371,8 +385,16 @@ public boolean isValidURI(String URI)
 	 */
 	private static void addVerifier(String element)
 	{
-		if("azimuth".equalsIgnoreCase(element))
-		{
+		if ("align-content".equalsIgnoreCase(element)) {
+			elementVerifiers.put(element, new CSSPropertyVerifier(Arrays.asList("flex-start", "flex-end", "center", "space-between", "space-around", "stretch"), ElementInfo.VISUALMEDIA));
+			allelementVerifiers.remove(element);
+		} else if("align-items".equalsIgnoreCase(element)) {
+			elementVerifiers.put(element, new CSSPropertyVerifier(Arrays.asList("flex-start", "flex-end", "center", "baseline", "stretch"), ElementInfo.VISUALMEDIA));
```

Comment:
Why are [`auto`, `last-baseline`, positions other than `flex-start`, `flex-end`, and `center`,](http://www.w3.org/TR/css3-align/#typedef-baseline-position) and the [`overflow-position`](http://www.w3.org/TR/css3-align/#typedef-overflow-position)s omitted?